services/ppac/src/main/java/app/coronawarn/datadonation/services/ppac/commons/PpaDataRequestConverter.java

```diff
@@ -38,7 +39,11 @@
           riskElement.getRiskLevelChangedComparedToPreviousSubmission(),
           TimeUtils.getLocalDateFor(riskElement.getMostRecentDateAtRiskLevel()),
           riskElement.getDateChangedComparedToPreviousSubmission(),
-          convertToUserMetadataDetails(userMetadata), technicalMetadata);
+          riskElement.getPtRiskLevelValue(), riskElement.getPtRiskLevelChangedComparedToPreviousSubmission(),
```

Comment:
According to the tech spec the logic is as follows: 
- if the the pt risk level is set to RISK_LEVEL_UNKOWN -> all PT* fields are set to null